sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -335,7 +335,7 @@ object FunctionRegistry {
     val df = clazz.getAnnotation(classOf[ExpressionDescription])
     if (df != null) {
       (name,
-        (new ExpressionInfo(clazz.getCanonicalName, name, df.usage(), df.extended()),
+        (new ExpressionInfo(clazz.getCanonicalName, name, df.usage(), df.extended().stripMargin),
```

Comment:
Add a new line character in a raw string does not work if we use the multi-line string. Thus, I added   `stripMargin` here for avoiding weird indenting.